browser-test/src/admin/admin_program_translation.test.ts

```diff
@@ -415,6 +440,20 @@ test.describe('Admin can manage program translations', () => {
         'Spanish block eligibility message',
       )
     })
-    // TODO: publish and verify the translation applicant-side
+
```

Comment:
**suggestion:** Could we add a test verifying what the applicant sees without a custom eligibility message set? Or is that already being tested?

Reply:
Happy to add one in a different PR since it's not really related to the eligibility msg feature.